lib/helpers/LanguageApiHelper.ts

```diff
@@ -57,4 +57,30 @@ export class LanguagesApiHelper {
   async delete(isoCode: string) {
     return await this.api.delete(this.api.baseUrl + '/umbraco/management/api/v1/language/' + isoCode);
   }
+
+  async ensureNameNotExists(name: string) {
+    const language = await this.getByName(name);
+    if (language !== null) {
+      await this.delete(language.isoCode);
+    }
+  }
+
+  async getAllLanguages() {
```

Comment:
The name of this method should be "getAll" since it belongs to LanguagesApiHelper. There is no need to define it as "getAllLanguages()."